src/screens/CreateCommunity/index.js

```diff
@@ -97,11 +97,18 @@ const CreateCommunity = () => {
     if (name && name.length >= MINIMUM_NAME_LENGTH && typeFetch === 'available') {
       const response = await submitCommunityName(name);
       if (response.success) {
-        navigation.replace('CreateCommunityCustomize', {
+        navigation.replace('ContactScreen', {
           isCreateCommunity: true,
           topicCommunityId: response.topic_id,
           topicCommunityName: name
         });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The navigation replacement to 'ContactScreen' seems fine, but it would be better if you could handle the case when `response.success` is false. Currently, there's no error handling for this scenario which might lead to unexpected behavior.
```diff
+      if (response.success) {
+        navigation.replace('ContactScreen', {
+          isCreateCommunity: true,
+          topicCommunityId: response.topic_id,
+          topicCommunityName: name
+        });
+      } else {
+        // Handle error here
+      }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->